tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/microprofiile/template/ConfigPropertyTestTemplate.java

```diff
@@ -63,7 +58,7 @@ public void prepareProject() {
 		te.save();
 		new WaitWhile(new EditorHasValidationMarkers(te));
 
-		convertToMavenProject(PROJECT_NAME, "war", false);
+		new AbstractMavenSWTBotTest(){}.convertToMavenProject(PROJECT_NAME, "war", false);
```

Comment:
what is this construct?

Reply:
I need to use method from abstract class -> anonymous class extending AbstractMavenSWTBotTest with empty {}, it means I do not need to implement/override anything from the AbstractMavenSWTBotTest class, I just need an instance to call a method.

Reply:
as agreed, it would be better to create new util class and use it here instead of using anonymous class initialization from the abstract class.

Reply:
Now I see it - I have created MavenProjectHelper class containing static method convertToMavenProject().